DataProc/CPass0/mergeMakeOCDB.byComponent.perStage.sh

```diff
@@ -262,6 +262,35 @@ main()
     echo "***********************" 2>&1 | tee -a ocdb.log
     echo aliroot -b -q "makeOCDB.C($run, \"$ocdb\", \"$defaultOCDB\", $detectorBitsQualityFlag)" 2>&1 | tee -a ocdb.log
     aliroot -b -q "makeOCDB.C($run, \"$ocdb\", \"$defaultOCDB\", $detectorBitsQualityFlag)" 2>&1 | tee -a ocdb.log
+    if [[ $ALIEN_JDL_CREATEOCDBARCHIVE ]]; then
```

Comment:
Environment variable `ALIEN_JDL_CREATEOCDBARCHIVE` (which would be `CreateOCDBArchive` on a JDL) is not defined on the Grid, so this part should not affect Grid jobs either.